tests/legacy-cli/e2e/tests/build/server-rendering/server-routes-preload-links.ts

```diff
@@ -215,6 +215,7 @@ async function spawnServer(): Promise<number> {
     ['run', 'serve:ssr:test-project'],
     /Node Express server listening on/,
     {
+      ...process.env,
```

Comment:
Should we just do this once in `execAndWaitForOutputToMatch` instead?

Reply:
Potentially, but I suspect there could be a few left where the inserting of `process.env` is maybe unintended. Let's leave as is for now and revisit when needed?